client/src/app/shared/page-not-found/page-not-found.component.html

```diff
@@ -1,3 +1,4 @@
 <p>
-  page-not-found works!
+  404!<br>
+  Sidan (vad du nu skrivit) fungerar ej! Prova www.i-portalen/
```

Comment:
Haha denna skulle ju kunna utvecklas lite men det funkar! 404 är inte heller bara för att folk skriver in urler som inte finns (om det var det du menade med skriver :) ) utan tror det ganska ofta beror på gamla länkar som inte uppdaterats. Det skulle t.ex. kunna vara en knapp på sidan som leder till en 404 då sidan den gick till har flyttats / tagits bort. @williamrimton